README.md

```diff
@@ -1,9 +1,55 @@
 [![Build Status](https://travis-ci.org/TREP-LABS/realdrip-backend.svg?branch=master)](https://travis-ci.org/TREP-LABS/realdrip-backend)
 [![Coverage Status](https://coveralls.io/repos/github/TREP-LABS/realdrip-backend/badge.svg?branch=master)](https://coveralls.io/github/TREP-LABS/realdrip-backend?branch=master)
-## RealDrip Backend
-`A node-js server using express-js
-`
+
+![GitHub contributors](https://img.shields.io/github/contributors/trep-labs/realdrip-backend)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/trep-labs/realdrip-backend)
+![GitHub pull requests](https://img.shields.io/github/issues-pr/trep-labs/realdrip-backend)
+![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed-raw/trep-labs/realdrip-backend)
+# RealDrip Backend
+ > [Learn about The Realdrip Device](https://treplabs.co/realdrip)
+
 #### The Setup integrated the following CI/CD tools
  - TravisCI
  - Coveralls.io
- - Jest
\ No newline at end of file
+ - Jest & Suppertest
+ #### Technologies
+ - Nodejs
+ - MongoDB
+ - Javascript
+
+## Important Dependacies
+ - [mongoose](https://mongoosejs.com/)
+ - [Jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken)
+ - [validator](https://www.npmjs.com/package/validator)
+ - [nodemailer](https://nodemailer.com/)
+ - [expressJs](https://expressjs.com/)
```

Comment:
Yes, these dependencies are important, however, they are no different from any other dependency in the `devDependency` section of `package.json` file e.g we have no authentication without `jwt`. Consequently, I think it's best we let the  `devDependency` section of `package.json` document the important dependency as opposed to adding them in the readme.

Reply:
Please can you be clearer on this.